packages/frontend/src/components/insight-list/insight-list.tsx

```diff
@@ -86,7 +86,7 @@ export const InsightList = ({
     );
   }
 
-  if (insightConnection.edges.length === 0) {
+  if (insightConnection.edges?.length === 0) {
```

Comment:
`edges` are not nullable, so not sure why this is needed?

Reply:
you're right, I'm not sure how I suddenly got an error from here, seems like `insightConnection` shouldn't be undefined either at this point.

But I couldn't replicate it anymore, so maybe I'll just remove it from here and create a ticket if it happens again